2021-06-05 19:10:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ограничение "GETBOOK_FK0" уже существует
Constraint "GETBOOK_FK0" already exists; SQL statement:
ALTER TABLE `getBook` ADD CONSTRAINT `getBook_fk0` FOREIGN KEY (`id_reader`) REFERENCES `readers`(`id_reader`); [90045-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:114)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at Main.Main.<init>(Main.java:74)
	at Main.Main.main(Main.java:428)
2021-06-05 19:12:40 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Николай/Desktop/ИРНИТУ/temp/library/Library/db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Николай/Desktop/ИРНИТУ/temp/library/Library/db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:270)
	at Main.Main.<init>(Main.java:72)
	at Main.Main.main(Main.java:428)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Николай/Desktop/ИРНИТУ/temp/library/Library/db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Николай/Desktop/ИРНИТУ/temp/library/Library/db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 26 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Николай/Desktop/ИРНИТУ/temp/library/Library/db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 20 more
2021-06-05 19:12:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ограничение "GETBOOK_FK0" уже существует
Constraint "GETBOOK_FK0" already exists; SQL statement:
ALTER TABLE `getBook` ADD CONSTRAINT `getBook_fk0` FOREIGN KEY (`id_reader`) REFERENCES `readers`(`id_reader`); [90045-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:114)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at Main.Main.<init>(Main.java:74)
	at Main.Main.main(Main.java:428)
2021-06-05 19:13:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ограничение "GETBOOK_FK0" уже существует
Constraint "GETBOOK_FK0" already exists; SQL statement:
ALTER TABLE `getBook` ADD CONSTRAINT `getBook_fk0` FOREIGN KEY (`id_reader`) REFERENCES `readers`(`id_reader`); [90045-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:114)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at Main.Main.<init>(Main.java:74)
	at Main.Main.main(Main.java:428)
2021-06-05 19:13:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ограничение "GETBOOK_FK0" уже существует
Constraint "GETBOOK_FK0" already exists; SQL statement:
ALTER TABLE `getBook` ADD CONSTRAINT `getBook_fk0` FOREIGN KEY (`id_reader`) REFERENCES `readers`(`id_reader`); [90045-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:114)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at Main.Main.<init>(Main.java:74)
	at Main.Main.main(Main.java:428)
2021-06-05 19:13:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ограничение "GETBOOK_FK0" уже существует
Constraint "GETBOOK_FK0" already exists; SQL statement:
ALTER TABLE `getBook` ADD CONSTRAINT `getBook_fk0` FOREIGN KEY (`id_reader`) REFERENCES `readers`(`id_reader`); [90045-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:114)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at Main.Main.<init>(Main.java:74)
	at Main.Main.main(Main.java:428)
2021-06-05 19:14:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ограничение "GETBOOK_FK0" уже существует
Constraint "GETBOOK_FK0" already exists; SQL statement:
ALTER TABLE `getBook` ADD CONSTRAINT `getBook_fk0` FOREIGN KEY (`id_reader`) REFERENCES `readers`(`id_reader`); [90045-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:114)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at Main.Main.<init>(Main.java:74)
	at Main.Main.main(Main.java:428)
